src/main/java/pl/javastart/task/FileOperator.java

```diff
@@ -0,0 +1,44 @@
+package pl.javastart.task;
+
+import java.io.*;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class FileOperator {
+
+    public Queue<Vehicle> readVehicles(String fileName) {
+        Queue<Vehicle> vehicles = new LinkedList<>();
+        try (var br = new BufferedReader(new FileReader(fileName))) {
+            String nextLine;
+            while ((nextLine = br.readLine()) != null) {
+                Vehicle vehicle = parseVehicle(nextLine);
+                vehicles.offer(vehicle);
+            }
+        } catch (IOException ex) {
+            throw new UncheckedIOException(ex);
+        }
+        return vehicles;
+    }
+
+    public void saveVehicles(Queue<Vehicle> vehicles, String fileName) {
+        try (var bw = new BufferedWriter(new FileWriter(fileName))) {
+            while (vehicles.peek() != null) {
+                String vehicleWithoutWhiteSpaces = vehicles.poll().toString();
```

Comment:
lepiej nie używać `toString() do serializacji. Ta metoda służy raczej do debugowania i mogłaby zostać opatrznie nadpisana. `Lombok` ma feature, który generuje `toString()` automatycznie dla wszystkich pól, więc dodanie pola mogłoby spowodować problemy z odczytem danych. Lepiej dodać dedykowaną metodę jak np `toCsv()`. Swoją drogą zamiast oddzielać spacjami, to oddzielaj przecinkami i rozszerzenie pliku ustaw jako `csv`.